src/PIL/TiffImagePlugin.py

```diff
@@ -1275,6 +1275,13 @@ def load(self) -> Image.core.PixelAccess | None:
             return self._load_libtiff()
         return super().load()
 
+    @property
+    def size(self) -> tuple[int, int]:
```

Comment:
This implementation is differ from https://github.com/uploadcare/pillow-simd/commit/1d77bfe9d939e208e3bb2b897a4209c0881eb9f1 since for me it looks less magical, but if you can't agree, I can copy the previous as is